src/ui/rcb-ui.svelte

```diff
@@ -1,323 +1,334 @@
-            <div class="rcb-ui-main">
-                <Shell>
-                    {#if state === "present-offer" && productDetails && purchaseOptionToUse}
-                        <StatePresentOffer productDetails={productDetails} purchaseOption={purchaseOptionToUse}/>
-                    {/if}
-                    {#if state === "present-offer" && !productDetails}
-                        <StateLoading/>
-                    {/if}
-                    {#if state === "needs-auth-info" || state === "processing-auth-info"}
-                        <StateNeedsAuthInfo
-                                onContinue={handleContinue}
-                                onClose={handleClose}
-                                processing={state === "processing-auth-info"}
-                                lastError={lastError}
-                        />
-                    {/if}
-                    {#if paymentInfoCollectionMetadata && (state === "needs-payment-info" || state === "polling-purchase-status")}
-                        <StateNeedsPaymentInfo
-                                {paymentInfoCollectionMetadata}
-                                onContinue={handleContinue}
-                                onClose={handleClose}
-                                onError={handleError}
-                                processing={state === "polling-purchase-status"}
-                        />
-                    {/if}
-                    {#if state === "loading"}
-                        <StateLoading/>
-                    {/if}
-                    {#if state === "error"}
-                        <StateError
-                                {brandingInfo}
-                                lastError={lastError ??
+          </Shell>
+        </div>
+      {/if}
+      <div class="rcb-ui-main">
+        <Shell>
+          {#if state === "present-offer" && productDetails && purchaseOptionToUse}
+            <StatePresentOffer
+              {productDetails}
+              purchaseOption={purchaseOptionToUse}
+            />
+          {/if}
+          {#if state === "present-offer" && !productDetails}
+            <StateLoading />
+          {/if}
+          {#if state === "needs-auth-info" || state === "processing-auth-info"}
+            <StateNeedsAuthInfo
+              onContinue={handleContinue}
+              onClose={handleClose}
+              processing={state === "processing-auth-info"}
+              {lastError}
+            />
+          {/if}
+          {#if paymentInfoCollectionMetadata && (state === "needs-payment-info" || state === "polling-purchase-status") && productDetails && purchaseOptionToUse}
+            <StateNeedsPaymentInfo
+              {paymentInfoCollectionMetadata}
+              onContinue={handleContinue}
+              onClose={handleClose}
+              onError={handleError}
+              processing={state === "polling-purchase-status"}
+              {productDetails}
+              {purchaseOptionToUse}
```

Comment:
These 2 lines are the only meaningful change in this file, the rest is just automatically fixed indentation